src/compiler/scanner.ts

```diff
@@ -1296,9 +1296,11 @@ namespace ts {
                     case CharacterCodes.exclamation:
                         if (text.charCodeAt(pos + 1) === CharacterCodes.equals) {
                             if (text.charCodeAt(pos + 2) === CharacterCodes.equals) {
-                                return pos += 3, token = SyntaxKind.ExclamationEqualsEqualsToken;

```

Comment:
i would rather we do not enable this rule either.

Reply:
#19780